src/components/asset-list/RecyclerAssetList2/profile-header/ProfileActionButtonsRow.tsx

```diff
@@ -59,21 +59,29 @@ export function ProfileActionButtonsRow() {
   }));
 
   if (!accentColorLoaded) return null;
+
+  const addCashEnabled = config.features.f2c_enabled;
+  const swapEnabled = config.features.swagg_enabled;
```

Comment:
😂 

should we change swapEnabled to swagEnabled or is this a typo? would love to turn the swag on